pages/common/git-checkout.md

```diff
@@ -4,12 +4,16 @@
 
 - Switch to another branch:
```

Comment:
suggestion - "Switch to an existing branch". To match with the next description.